### ISSUE TITLE
UnboundLocalError: local variable 'SL_m' referenced before assignment

### ISSUE THREAD

**Issue by apolionl** [qtm-iisc/Twister#7]
Good morning.

I have been testing the code in other ranges where I am suppose to get a twst angle. Then I found an error

##
Traceback (most recent call last):
  File "/solid/Twister/SRC/get_angle.py", line 74, in <module>
Note: Since strain_tensor_vector is True, only the top layer will be strained regardless of strain_layer
    deep_search(a1,a2,b1,b2,theta_i,theta_f,dth,mismatch,strain_per,strain_layer,fix_ang,f_ang,range_nm_l,range_nm_u,th_er,alat_a,alat_b,nba_a,nba_b,plot,strain_tv,deepsearch,f_thresh)
  File "/solid/Twister/SRC/get_angle_funcs.py", line 807, in deep_search
    mism_ang,min_str_list,SL,SL_m,total_atoms,SL_ang,Strain_avg,strd_uv,p,u,new_ag_list,new_uv,tem_mis,alat_a,alat_b,ind_list,total_atom_list=clt(a1,a2,b1,b2,theta_i,theta_f,dth,mismatch,strain_per,strain_layer,fix_ang,f_ang,range_nm_l,range_nm_u,th_er,alat_a,alat_b,nba_a,nba_b,plot,strain_tv,deepsearch)
  File "/solid/Twister/SRC/get_angle_funcs.py", line 585, in clt
    SL_m=comm.bcast(SL_m,root=0)
UnboundLocalError: local variable 'SL_m' referenced before assignment

##

In short, the error now occurs in the line 585

I have not tied to fix it yet, cause I do not know what the value should be. , So if you could please let me know your thoughts, I would appreciate it.

**Comment by apolionl** [comment]
I found the error. 
It was a mistype in the input file. So I am closing this issue.
